boards/cargo/panic_abort_tests.toml

```diff
@@ -0,0 +1,8 @@
+# Licensed under the Apache License, Version 2.0 or the MIT License.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+# Copyright Tock Contributors 2024.
+
+[unstable]
+# Build test crates with the `panic = "abort"` strategy. Tests use `unwind` by
+# default which causes incompatability with `core` built via `-Zbuild-std`.
+panic-abort-tests = true
```

Comment:
It would be nice to have a comment about _when_ a board might want this, and when they wouldn't.

Reply:
If and only if they have tests